### ISSUE TITLE
Support fine-tuning for chatgpt 3.5

### ISSUE THREAD

**Issue by INF800** [anarchy-ai/LLM-VM#132]
We can fine-tune gpt 3.5 i.e chat_gpt according to [official blog](https://openai.com/blog/gpt-3-5-turbo-fine-tuning-and-api-updates) and [docs](https://platform.openai.com/docs/guides/fine-tuning)

So, isn't it a good idea to remove the below line and make sure fine-tuning happens?
https://github.com/anarchy-ai/LLM-VM/blob/3b4ade5a40c5160b71642373ae19ba8391066f84/src/llm_vm/onsite_llm.py#L444

Note: `gpt-3.5-turbo-0301` is used in `generate` method but `gpt-3.5-turbo-0613` supports fine-tuning. Suffix is a different number.

**Comment by INF800** [comment]
Can send a PR for this if required. Please let me know.

**Comment by mmirman** [comment]
It's probably quite a bit more than just removing that line, but this is a good first issue!   I think 3.5 has different formats than 3 so the PR should have tests in it :-)  But thanks for submitting the issue!

**Comment by INF800** [comment]
Makes sense. Will make required changes and send PR with tests. I have a small doubt - do you use pytest for writing tests or some other testing tool? Asking because I don't see pytest being used in tests/ directory.

**Comment by mmirman** [comment]
No pytest yet - best to make a ticket for it!

**Comment by INF800** [comment]
> No pytest yet - best to make a ticket for it!

Opened #133

**Comment by bilal-aamer** [comment]
Will attempt to close this
